packages/cli-polaris/project.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Same here, once we [merge this PR](https://github.com/Shopify/cli/pull/353) we'll have to add a `targets` section here with all the tasks.